services/providers/wanaku-provider-file/src/main/resources/application.properties

```diff
@@ -30,4 +30,7 @@ wanaku.service.provider.service.defaults.idempotent=false
 # valkey.port=6379
 
 # Timeout
-# valkey.timeout=10
\ No newline at end of file
+# valkey.timeout=10
+
+#quarkus.mongodb.connection-string = mongodb://localhost:27017
+#wanaku.persistence.mongodb = true
```

Comment:
I guess this is out of scope, as downstream services should not interact w/ the persistence layer.